Rehlate Landing Page/index.html

```diff
@@ -0,0 +1,644 @@
+                          
+                           <div class="container">
+                            <h4 class="darkBlack">دبي</h4>
+                            <p class="Gray-100">3 أيام بتكلفة 2,500 دولار</p>
+                           </div>
+                          
+                        </div>
+                    </div>
+                </div>
+                <div class="section-travel-item">
+                    <div class="section-item-img p-relative">
+                       <img src="./assets/images/cairo-img.svg" alt="cairo-img"/>
+                        <div class="section-travel-content primaryColor-bg    p-absolute bottom-30 ">
+                          
+                           <div class="container">
+                            <h4 class="darkBlack">القاهرة</h4>
+                            <p class="Gray-100">3 أيام بتكلفة 1,250 دولار</p>
+                           </div>
+                          
+                        </div>
+                    </div>
+                </div>
+                <div class="section-travel-item">
+                    <div class="section-item-img p-relative">
+                       <img src="./assets/images/riyadh-img.jpg" alt="riyadh-img "/>
+                        <div class="section-travel-content primaryColor-bg    p-absolute bottom-30 ">
+                          
+                           <div class="container">
+                            <h4 class="darkBlack">الرياض</h4>
+                            <p class="Gray-100">2 أيام بتكلفة 1,500 دولار</p>
+                           </div>
+                          
+                        </div>
+                    </div>
+                </div>
+                     
+            </div>
+        </div>
+    </section>
+    <section id="Trips" class="trips">
+        <div class="container text-center">
+            <p class="GrayColor">تمتع في رحلتك</p>
+            <h2 class="darkBlack">مجموعة من خدماتنا</h2>
+            <div class="d-flex trip padding-top-100 padding-bottom-100  text-center radius-25 CandyGrape-100-bg gap-25">
+                
+                    
+                <div class="section-trips">
+                  
+                    <div class="section-trip-img">
+                        <img src="./assets/images/services-comp.png" alt="Tourguide"/>
+                    </div>
+                    <div class="section-trips-content">
+                        <h4>دليل سياحي</h4>
+                        <p>نقدم لك خدمات الإرشاد السياحي لبناء
+                            خطط سفر مميزة.</p>
+                    </div>
+                </div>
+                <div class="section-trips ">
+                    
+                    <div class="section-trip-img">
+                        <img src="./assets/images/services-car.png" alt="Tourguide"/>
+                    </div>
+                    <div class="section-trips-content">
```

Comment:
This section is missing the purple arrow

Reply:
iam sorry for forget it i made it the purple arrow .